### PR TITLE
Add rate computation without GLOBES

### PR DESCRIPTION
**Pull request by soso128**
Since someone in my group didn't want to install GLoBES, I modified the Snowglobes interface to allow to compute the expected rates using only the SNOwGLoBES detector configurations and cross-sections. Here, the rates are computed for a perfect detector (no smearing, 100% efficiencies). I added options in snowglobles.simulate and snowglobes.collate in order to choose to compute this rate if needed.
Running this code for IceCUBE and for the Nakazato 2013 model with 30 bins (default options in scripts/TimeSeries.py), I found that the discrepancies between the new and old unsmeared rates were of at most 0.4%.

### CODE REVIEW

**Review by Sheshuk — COMMENTED** [review]
It's nice to have a separate rates calculation! I think it should also be possible to include also the smearing effects from SNOwGLoBES - because basically GLoBES is used to multiply vector by matrix and for interpolation. 

However I think this is strictly speaking not SNOwGLoBES. So I suggest to separate this functionality in a new class (perhaps a child of `SNOwGLoBES`) - it saves a code several `if`s? will be less misleading and also will not complicate the SNOwGLoBES class usage with additional flags

**Review by Sheshuk — CHANGES_REQUESTED** [review]
Also, when doing rate calculations it doesn't make sense to run in parallel threads (because python will do them sequentially anyway), so I suggest to:
1. Subclass `SNOwGLoBES`
2. Reimplement `run(self, flux_files, detector, material)` method using your code from `_compute_rates`.

**Comment by soso128** [comment]
Hi, thank you for the suggestion! I now implemented a separate class with its own run function.

File: python/snewpy/snowglobes_interface.py [hunk 1]
```diff
@@ -218,8 +218,6 @@ class Runner:
     def __post_init__(self):
         self.channels=self.sng.channels[self.material]
         self.binning=self.sng.binning[self.material]
-        self.efficiency=self.sng.efficiencies[self.detector]
```
Comment (Sheshuk) [inline python/snewpy/snowglobes_interface.py:221]: Is there a reason for removing `self.efficiency` and `self.smearing` initialization? 
Probably they need to be restored, for the Runner class to work

File: python/snewpy/snowglobes_interface.py [hunk 2]
```diff
@@ -284,5 +282,109 @@ def run(self):
             return tables
         else:
             raise RuntimeError('SNOwGLoBES run failed:\n'+stderr)
+
+class SimpleRate(SNOwGLoBES):
+    def __init__(self, base_dir:Path=''):
+        """ SNOwGLoBES interface
```
Comment (Sheshuk) [inline python/snewpy/snowglobes_interface.py:288]: Could you put the description of the rate calculation in the docstring here?

File: python/snewpy/snowglobes_interface.py [hunk 3]
```diff
@@ -284,5 +282,109 @@ def run(self):
             return tables
         else:
             raise RuntimeError('SNOwGLoBES run failed:\n'+stderr)
+
+class SimpleRate(SNOwGLoBES):
```
Comment (Sheshuk) [inline python/snewpy/snowglobes_interface.py:286]: Oh, if you don't reuse the `__init__` method from `SNOwGLoBES`, there is no reason to subclass SNOwGLoBES Sorry, I didn't realize in the first place, when I suggested you to use subclass. In the current implementation, these two classes don't have any shared methods, so subclassing has no effect.

**Review by Sheshuk — CHANGES_REQUESTED** [review]
Thanks for implementing my suggestions, this looks good! If you address the comments, I think we'll be ready to merge.

File: python/snewpy/snowglobes_interface.py (hunk 3 above)
Comment (soso128) [reply python/snewpy/snowglobes_interface.py:286]: Actually I am using the _load_detectors and _load_channels methods of the SNOwGLoBES  class. Do you think I should just add them to the SimpleRate class and drop the inheritance?

File: python/snewpy/snowglobes_interface.py (hunk 1 above)
Comment (soso128) [reply python/snewpy/snowglobes_interface.py:221]: This was an accident, I'll add it back

File: python/snewpy/snowglobes_interface.py (hunk 3 above)
Comment (Sheshuk) [reply python/snewpy/snowglobes_interface.py:286]: You're right, sorry, I missed it. Then it's good and completely adequate as it is

**Comment by soso128** [comment]
I have put the "self.efficiency=self.sng.efficiencies[self.detector]" line back and moved the docstring

**Review by Sheshuk — APPROVED** [review]
Looks good! Thanks for implementing this

**Comment by Sheshuk** [comment]
Oops, I overlooked that `Runner.smearing` is still missing...

**Comment by soso128** [comment]
Oops, sorry, I have just put it back

**Comment by Sheshuk** [comment]
Too late - I already merged it :( Created #174 with a hotfix

**Comment by soso128** [comment]
Ok, thanks a lot and sorry for the mistake!

**Pull request merged**